provider.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2023 Kapeta Inc.
 // SPDX-License-Identifier: MIT
 
-package sdkgoconfig
+package kapeta
```

Comment:
Not sure I'm convinced this is a good name, since we have a ton of kapeta 'packages'  so unless we feel like this is the main main package... :)

Reply:
I think it is - alternative would be "config" ? sdkgoconfig contains a lot of not needed information?

Reply:
Changed to config